Automattic/newspack-listings
# feat: use post categories and tags for all listing post types

### All Submissions:






### Changes proposed in this Pull Request:



Deprecates the custom category and tag taxonomies that apply only to listing CPTs, in favor of regular post categories and tags. This will make it much easier to create relationships between listing and other types of site content.

Note: the `epic/phase-2` branch is currently a direct clone of `master`. I'll open all Phase 2 PRs against that branch so we can work on multiple features in parallel, then merge the epic branch into master when Phase 2 is release-ready.

Closes #32.

### How to test the changes in this Pull Request:

1. In WP admin, confirm that the **Listings > Categories** and **Listings > Tags** menu items now redirect to the post categories and tags.
2. Edit a listing of any type. Confirm that when searching for categories and tags, the suggestions are existing post categories and tags. Apply some categories and tags to several listing posts.
3. Create a query-mode curated list block in any page or post. Add some categories and tags to your query terms. Set the display options to show categories and tags. Confirm that it populates the listing posts you expect, with the matching terms, in both the editor and on the front-end.
4. Create another curated list block in "specific listings" mode. Set the display options to show categories and tags. Add several listings of various types and confirm that you see the correct categories and tags in both the editor and on the front-end.
5. View the front-end of an individual listing with categories and tags. Confirm that the categories shown on the singular page display correctly and link to the post archive pages for those categories (for now—in the future, these will need to link to archive pages for the listings, instead).
6. Confirm that on post category and tag archives, no listings appear. In the future, we will establish listings-specific archives to show the listings posts.

### Other information:
